### ISSUE TITLE
Clicking the discount tag in the grid no longer sends the user to its promo page?

### ISSUE THREAD

**Issue by Auster-South-Anemoi** [ghorint2t/scripts#15]
Title.

**Comment by ghorint2t** [comment]
Yes, that is the unfortunate side effect of switching over to the new API. The old one doesn't have tags, the new one doesn't have promo ids. Technically I could query both of them, but not sure this little thing is worth doubling size of download data (and also wait time before the grid loads).

**Comment by DawEdhel** [comment]
> Technically I could query both of them, but not sure this little thing is worth doubling size of download data (and also wait time before the grid loads).

Maybe instead of querying it on page load, rather implement a new action for an user to click and find, if available?
This way it will be a minimum extra load for the script at a cost of extra click on an user's side.

**Comment by ghorint2t** [comment]
OK, I finally got around to doing it. I've added a button in the Discount column menu that will do just that - get the promo IDs from old API and create links from them.
